cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -174,7 +174,7 @@ private static File createLocalDir(CConfiguration cConf, String dirName) {
       File credentialsDir = new File(localDirStr, dirName);
 
       try {
-        java.nio.file.Files.createDirectories(credentialsDir.toPath(), FileUtils.OWNER_ONLY_RWX);
```

Comment:
There must be a reason this used to be created with that set of permissions. What breaks if we remove that?